docs/source/supported/bridges2.rst

```diff
@@ -1,4 +1,113 @@
-=============
-PSC Bridges-2
-=============
+====================
+Bridges2 (ACCESS)
+====================
 
+Platform user guide
+===================
+
+https://www.psc.edu/resources/bridges-2/user-guide-2-2/
+
+General description
+===================
+
+* Resource manager - ``SLURM``
+* Launch methods (per platform ID)
+
+  * ``access.bridges2*`` - ``MPIRUN``
+
+* Configuration per node (per platform ID)
+
+  * ``access.bridges2.RM`` (50 nodes)
+
+    * 128 CPU cores (1 thread per core)
+    * 256GB GiB of memory
+
```

Comment:
we have only one configuration record for Bridges2, but we can expand it with queues (just in a way not to confuse users)

For example:
```
* Configuration per node (per platform ID and per queue)

   * ``access.bridges2``

     * queue: ``RM`` (50 nodes)

       * 128 CPU cores (1 thread per core)
       * 256GB GiB of memory
```

(*) we don't have such platform ID `access.bridges2.RM` - that what would be confusing (for Delta it was created a configuration per queue)